cs50/cli.md

```diff
@@ -1,3 +1,3 @@
-# `cs50/cli`
+# cs50/cli
```

Comment:
The backtick for titles is removed due to issue: https://github.com/ABD-01/furo50/issues/11